homeassistant/components/mystrom/light.py

```diff
@@ -69,12 +71,31 @@ def __init__(self, bulb, name):
         self._brightness = 0
         self._color_h = 0
         self._color_s = 0
+        self._mac = mac
+        self.bulb_type = None
+        self.firmware = None
 
     @property
     def name(self):
         """Return the display name of this light."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return self._mac
+
+    @property
+    def device_info(self):
+        """Return device specific attributes."""
+        return {
+            "identifiers": {(DOMAIN, self.unique_id)},
+            "name": self._name,
+            "manufacturer": DOMAIN,
+            "model": self.bulb_type,
```

Comment:
This would be `rgblamp`. I doubt it's called like that on their site. Maybe RGB Lamp or even something completely else?

Reply:
No, it's not but that's what the API is returning. I will change that.